### PR TITLE
Account for whitespace abnormalities in GPS coord parsing

### PR DESCRIPTION
**Pull request by mjriley**
## Technical Summary
Associated bug: https://dimagi-dev.atlassian.net/browse/SAAS-14313

### Safety story
Unit tests cover the scenarios that would fix the sentry error in the above ticket. Because I had difficulty working with GPS data locally, this was not tested locally.

### Automated test coverage

new test suite created in corehq.apps.export.tests.test_new

### QA Plan

No QA necessary

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/export/models/new.py [hunk 1]
```diff
@@ -2735,26 +2735,38 @@ def get_headers(self, split_column=False):
...
+        values = [EMPTY_VALUE] * NUM_VALUES
+        if not isinstance(coord_string, str):
             return values
 
-        for index, coordinate in enumerate(value.split(' ')):
+        for index, coordinate in enumerate(coord_string.split()):
+            # NOTE: Unclear if the intention here is to support situations where only the lat/lng are supplied,
+            # or if we really want to allow just specifying just lat, or lat/lng/alt as valid.
+            # I think it's likely we only want to support the lat/lng situation, but leaving as is
+            # in the event that this behavior is relied upon somewhere
```
Comment (esoergel) [inline corehq/apps/export/models/new.py:2765]: No idea whether lt/lng/alt is valid.

**Review by esoergel — APPROVED** [review]
I don't see any reason to be concerned about switching from `.split(' ')` to `.split()`, that seems fine to me (and I'd be supportive of making that change to `GeoPoint` too if used here).

I am tempted to suggest using `GeoPoint.from_string` in the interest of unifying implementations, but that does apply much stricter constraints (it coerces to decimals and ensures they are within valid ranges), so there's always a reasonable chance it'd break existing workflows.  Maybe useful if we need to turn these into decimal types in excel exports, but probably not worth the risk for untyped values.

**Review by snopoke — APPROVED** [review]

File: corehq/apps/export/models/new.py (hunk 1 above)
Comment (snopoke) [reply corehq/apps/export/models/new.py:2765]: I don't have any more context on whether we want to allow those or not but I don't think it hurts, especially in exports.

File: corehq/apps/export/models/new.py [hunk 2]
```diff
@@ -2735,26 +2735,38 @@ def get_headers(self, split_column=False):
...
-            return value
+            return coord_string
 
-        if value == MISSING_VALUE:
-            return [MISSING_VALUE] * 4
+        return self.extract_coordinate_array(coord_string)
 
-        values = [EMPTY_VALUE] * 4
+    @classmethod
+    def extract_coordinate_array(cls, coord_string):
```
Comment (kaapstorm) [inline corehq/apps/export/models/new.py:2751]: It looks like `cls` is unused. You could go for
```suggestion
    @staticmethod
    def extract_coordinate_array(coord_string):
```

**Review by kaapstorm — APPROVED** [review]

**Pull request merged**